Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Logic/TargetedSequencer.h

```diff
@@ -27,7 +27,12 @@ namespace ScriptCanvas
 
             public:
 
-                TargetedSequencer();
+                TargetedSequencer()
+                    : Node()
+                    , m_numOutputs(0)
+                {}
```

Comment:
We have moving functions from the header to cpp file in order to reduce our large compile times.
https://github.com/o3de/sig-core/issues/53

I am assuming you have moved this constructor to the header file, because this class is access outside of the ScriptCanvas gem, but this function doesn't export it's symbols?

Reply:
Correct, this helped when working through issues with static registration, while I was able to fix them more properly by making sure the header file is included, in some cases I left these changes to avoid spending too much time chasing the same problem. Once this change is in I will do a cleanup pass and try to rectify this, but at this time it works and passes AR for all platforms, it will be simpler to work on these issues once this change is in place.